### PR TITLE
Add Beta Endpoints

### PR DESCRIPTION
**Pull request by thehappydinoa**
# Description

Added all Beta ASM API Endpoints as well as miscellaneous updates. 

# Changes

- Added `Beta` ASM API class
- Updated the `Events` class to be called `Logbook`
- Added the Object Storage Asset type
- Updated and tested various time parsing utils

### CODE REVIEW

File: censys/common/utils.py [hunk 1]
```diff
@@ -17,3 +17,17 @@ def format_rfc3339(time: Datetime) -> str:
     if isinstance(time, (datetime.date, datetime.datetime)):
         time = time.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
     return time
+
+
+def format_iso8601(time: Datetime) -> str:
```
Comment (dcaponi-censys) [inline censys/common/utils.py:22]: surprised this isnt part of the regular ol datetime library somewhere

Comment (thehappydinoa) [reply censys/common/utils.py:22]: Yeah, we don't use the full ISO 8601, otherwise we could use `date.isoformat()`

File: docs/usage-asm.rst [hunk 2]
```diff
@@ -202,40 +203,40 @@ Below we show examples for **creating a logbook cursor** for retrieving filtered
...
 
 Below we show examples for **getting logbook events.**
 
 .. code:: python
 
-    from censys.asm import Events
+    from censys.asm import Logbook
 
-    e = Events()
+    l = Logbook()
```
Comment (dcaponi-censys) [inline docs/usage-asm.rst:232]: ooff single letter names. i guess if thats the way, thats the way

**Review by dcaponi-censys — APPROVED** [review]
looks sane

**Pull request merged**